scripts/utils.js

```diff
@@ -191,14 +191,20 @@ const updateDenoImportMap = async (importMapPath, nextVersion) => {
 /**
  * @param {string} nextVersion
  */
-async function incrementRemixVersion(nextVersion) {
+async function incrementRemixVersion(nextVersion, syncLockFile = false) {
   // Update version numbers in package.json for all packages
   await updateRemixVersion("remix", nextVersion);
   await updateRemixVersion("create-remix", nextVersion);
   for (let name of remixPackages.all) {
     await updateRemixVersion(`remix-${name}`, nextVersion);
   }
 
+  // Sync up the pnpm-lock.yaml file with the new experimental version
+  if (syncLockFile) {
+    console.log(chalk.green("  Syncing pnpm lockfile..."));
+    execSync("pnpm install --no-frozen-lockfile");
```

Comment:
Need to do this with the version bump otherwise the CI run will fail on `pnpm install --frozen-lockfile`